libnet/include/stamp-h.in

```diff
@@ -1 +0,0 @@
-timestamp
```

Comment:
HAHA I thought Makefile.am.common was confusing until I saw this file. 
I read a little about it, and it turns out that it purpose is to avoids unnecessary recompilation of config.h.in. 
https://www.gnu.org/software/autoconf/manual/autoconf-2.63/html_node/Automatic-Remaking.html

I temporarily removed it too and during build, when make enters the include directory it does seem like it's doing some extra work:

```
Making all in include
make[1]: Entering directory `/mnt/c/Users/Ali/projects/libnet/libnet/include'
(CDPATH="${ZSH_VERSION+.}:" && cd .. && /bin/bash /mnt/c/Users/Ali/projects/libnet/libnet/missing autoheader)
rm -f stamp-h1
touch config.h.in
cd .. && /bin/bash ./config.status include/config.h
config.status: creating include/config.h
config.status: include/config.h is unchanged
make  all-recursive
```